### ISSUE TITLE
Boxen's hombrew calling Formula.sha1 which is deprecated 

### ISSUE THREAD

**Issue by kaylacohn** [boxen/puppet-homebrew#96]
These deprecations caused hub not to install on my comp when I ran boxen. I can install this manually with `brew install hub` but I think these deprecation warnings should probably be looked at to see if anything needs to be taken care of.  

```
Notice: /Stage[main]/Hub/Package[boxen/brews/hub]/ensure: ensure changed '2.2.1' to 'Warning: Calling Formula.sha1 is deprecated!
Use Formula.sha256 instead.
/usr/local/Library/Taps/boxen/homebrew-brews/hub.rb:6:in `<class:Hub>'
Please report this to the boxen/brews tap!

Warning: Calling SoftwareSpec#sha1 is deprecated!
Use SoftwareSpec#sha256 instead.
/usr/local/Library/Taps/boxen/homebrew-brews/hub.rb:6:in `<class:Hub>'
Please report this to the boxen/brews tap!

Warning: Calling Resource#sha1 is deprecated!
Use Resource#sha256 instead.
/usr/local/Library/Taps/boxen/homebrew-brews/hub.rb:6:in `<class:Hub>'
Please report this to the boxen/brews tap!
```

**Comment by KushalP** [comment]
Seeing something similar with `dnsmasq` from `boxen/brews`:

```
Warning: Calling Formula.sha1 is deprecated!
Use Formula.sha256 instead.
/opt/boxen/homebrew/Library/Taps/boxen/homebrew-brews/dnsmasq.rb:6:in `<class:Dnsmasq>'
Please report this to the boxen/brews tap!

Warning: Calling SoftwareSpec#sha1 is deprecated!
Use SoftwareSpec#sha256 instead.
/opt/boxen/homebrew/Library/Taps/boxen/homebrew-brews/dnsmasq.rb:6:in `<class:Dnsmasq>'
Please report this to the boxen/brews tap!

Warning: Calling Resource#sha1 is deprecated!
Use Resource#sha256 instead.
/opt/boxen/homebrew/Library/Taps/boxen/homebrew-brews/dnsmasq.rb:6:in `<class:Dnsmasq>'
Please report this to the boxen/brews tap!
```

**Comment by jacobbednarz** [comment]
@kaylacohn this has been fixed in boxen/puppet-hub#16

@KushalP there is an issue open over at boxen/puppet-dnsmasq#20 to get this fixed.
